index.html

```diff
@@ -0,0 +1,201 @@
+<!DOCTYPE html>
+
+<html>
+
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>Software Developer Training School</title>
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="style.css" />
+</head>
+
+<body class="homepage">
+    <section class="intro">
+        <div class="intro-sect">
+            <header class="nav-bar">
+                <div class="mobile-nav">
+                    <img class="hamburger" src="images/mobile_menu.png" alt="hamburger menu button" />
+                </div>
+                <div class="desktop-nav mobile-toggle">
+                    <img class="close-icon" src="images/close_mobile_menu.png" alt="close menu button" />
+                    <div class="logo">
+                        <a href="index.html"><img class="logo" src="images/my_logo.png" alt="logo image" /></a>
+                    </div>
+                    <div class="nav-container">
+                        <ul class="items">
+                            <a href="about.html">
+                                <li class="nav-item">About</li>
+                            </a>
+                            <a href="">
+                                <li class="nav-item">Programs</li>
+                            </a>
+                            <a href="">
+                                <li class="nav-item">Join Us</li>
+                            </a>
+                            <a href="">
+                                <li class="nav-item">Sponsors</li>
+                            </a>
+                            <a href="">
+                                <li class="nav-item">Info</li>
+                            </a>
+                        </ul>
+                        <button class="nav-btn">Upcoming</button>
+                    </div>
```

Comment:
Your nav is looking really good! Please be sure to use the `nav` tag for these links. Keep in mind that it should still be inside your `header` 😉. By doing this, you will improve your code navigation and accessibility.
[Learn more here.](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav)